src/main/java/edu/wisc/my/messages/service/ExpiredMessagePredicate.java

```diff
@@ -27,7 +28,8 @@ public boolean test(Message message) {
       new IsoDateTimeStringBeforePredicate(when);
 
     try {
-      return beforeWhen.test(message.getExpireDate());
+      MessageFilter filter = message.getFilter();
+      return beforeWhen.test(filter.getExpireDate());
```

Comment:
Pretty sure not handling `null` `filter` here is not okay.

I get 
![no-messages](https://user-images.githubusercontent.com/952283/36695088-d82fae6c-1b05-11e8-9179-152a13f09d87.png)

but expected that the message with no filtering would be included.

Reply:
In line 12, there's a comment --- Throws RuntimeException on evaluating null message.

What's the expected behavior on the null?

Reply:
Fixed.